### DISCUSSION TITLE
app crashes when trying to play a video

### DISCUSSION THREAD

**Discussion by jorgetherod** [qnblackcat/uYouPlus#1374] [General]
i am using a iphone 13 pro max on ios version 17.4.1 i need help on what to do to fix this issue

**Comment by Birdbang** [comment]
same issue happens on my iPhone IOS 17.4.1 with uYouPlus v19.08.2-3.0.3 today.

**Reply by abacnus** [reply]
I wonder if the latest merged PR fix this issue? https://github.com/qnblackcat/uYouPlus/pull/1365

**Reply by abacnus** [reply]
building the IPA to see..

**Reply by abacnus** [reply]
ah, see https://github.com/Galactic-Dev/iSponsorBlock/issues/110 
it seems that workaround is to use 19.02.01 to build

**Comment by GreyBoyTinker** [comment]
Same here, altough it was working fine yesterday and I'm not on 17.4.1 yet. Should I update and see what happens for science?

**Comment by Birdbang** [comment]
A workaround is disable iSponsorBlock. [https://github.com/arichornlover/uYouEnhanced/issues/484](url)

**Reply by ptu1710** [reply]
Thanks! It really works!!!
